File: cpp/bc1101.cpp
#include <iostream>

using namespace std;

int main()
{
	int m, n;
	
	while(1)
	{
		int sum = 0;
		
		cin >> m >> n;
		
		if(m <= 0 || n <= 0) break;
		else if(n < m)
		{
			int aux = m;
			m = n;
			n = aux;
		}
		
		for(int i = m; i <= n; i++)
		{
			sum += i;
			cout << i << " ";
		}
		cout << "Sum=" << sum << endl;
	}
}

File: python/bc1101.py
while True:
    sumT = 0

    pVal = input().split()

    fVal = int(pVal[0])
    sVal = int(pVal[1])

    if fVal <= 0 or sVal <= 0: break

    if sVal < fVal:
        aux = fVal
        fVal = sVal
        sVal = aux

    for c in range(fVal, sVal+1):
        print(f"{c}", end=" ")
        sumT += c
    print(f"Sum={sumT}")